playbooks/service-provision.yaml

```diff
@@ -127,9 +127,23 @@
         metadata:
           labels:
             demo.redhat.com/cfme: "true"
+          annotations:
+            demo.redhat.com/last-login: "{{ lookup('pipe', 'date +%Y-%m-%dT%H:%MZ') }}"
           name: "{{ cloudforms_username }}"
     register: r_user_created
 
+  - name: Add last-login annotation {{ cloudforms_username }}
+    when: r_user.resources | length > 0
+    k8s:
+      kubeconfig: "{{ kubeconfig }}"
+      definition:
+        apiVersion: user.openshift.io/v1
+        kind: User
+        metadata:
+          annotations:
+            demo.redhat.com/last-login: "{{ lookup('pipe', 'date +%Y-%m-%dT%H:%MZ') }}"
+          name: "{{ cloudforms_username }}"
+
```

Comment:
I recommend a bit different to isolate the fields patched from the resource apiVersion/kind/name:
```
  - name: Update last-login annotation {{ cloudforms_username }}
    when: r_user.resources | length > 0
    k8s:
      api_version: user.openshift.io/v1
      kind: User
      kubeconfig: "{{ kubeconfig }}"
      name: "{{ cloudforms_username }}"
      definition:
        metadata:
          annotations:
            demo.redhat.com/last-login: "{{ lookup('pipe', 'date +%Y-%m-%dT%H:%MZ') }}"
```